tests/test_idpi/test_regrid.py

```diff
@@ -0,0 +1,26 @@
+# Third-party
+from numpy.testing import assert_allclose
+
+# First-party
+from idpi import grib_decoder
+from idpi.operators import regrid
+from idpi.operators.hzerocl import fhzerocl
+
+
+def test_regrid(data_dir, fieldextra):
```

Comment:
Is this a regression test against fieldextra?
If so, maybe add it in a doc string.

Reply:
With that wider scope, I agree, such a comment is unnecessary.